stubs/wordpress.php

```diff
@@ -36,7 +36,8 @@
     */
 
     'exclude' => [
-        //  'path/to/directory-or-file'
+        'wp-admin',
```

Comment:
This should be on the default preset, defined on the core of php-insights.

Reply:
I'm having an error on this file

```php
/wp-admin/options-permalink.php
PHP Fatal error:  Uncaught TypeError: Argument 2 passed to SlevomatCodingStandard\Helpers\FunctionHelper::getParametersTypeHints() must be of the type integer, null given, called in /Volumes/Projects/www/xxx.com.dev.synot.io/site/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/TypeHints/DisallowArrayTypeHintSyntaxSniff.php on line 150 and defined in /Volumes/Projects/www/xxx.com.dev.synot.io/site/vendor/slevomat/coding-standard/SlevomatCodingStandard/Helpers/FunctionHelper.php:143
Stack trace:
```

Which exclusion should it be?

Reply:
Probably html code :(

Reply:
Alright, I was testing on our projects and it does require some extra phpinsights.php to be able to run, I'm gonna test on a more common WordPress installation, also bring the test for this new preset